hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -376,6 +376,11 @@ public void start(OzoneConfiguration configuration) {
    */
   public void startSnapshotSstFilteringService(OzoneConfiguration conf) {
     if (isSstFilteringSvcEnabled()) {
+      if (isDefragSvcEnabled()) {
+        LOG.info("SstFilteringService is disabled (despite config intends to enable it) " +
+            "because SnapshotDefragService is enabled. Defrag effectively performs filtering already.");
```

Comment:
maybe add another line when isSstFilteringSvcEnabled(), log "SstFilteringService to be removed in a future release" (if my memory serves me right, we discussed about it before)